spark/src/main/scala/ai/chronon/spark/Extensions.scala

```diff
@@ -325,4 +329,12 @@ object Extensions {
       }
     }
   }
+
+  implicit class TupleToJMapOps[K, V](tuples: Iterator[(K, V)]) {
+    def toJMap: util.Map[K, V] = {
```

Comment:
Will it be more performant than the Scala Map?

Reply:
Scala map somehow is not serializable :-/